client/src/components/DisplayTable.scss

```diff
@@ -3,6 +3,9 @@
 table.display-table {
   width: 99%; // avoid annoying 1px scroll caused by border
 }
+.total-row td {
+  font-weight: $heavyFontWeight !important;
+}
```

Comment:
So it looks like we have a `table-row` class defined in `table.scss` (along with a bunch of other styles that I think this table indirectly relies on). This selector is too generic, it's going to hit other `total-row` elements that are based on `table.scss` and weren't intended to have this style. It's more of a pain to write, but you'll need a specific selector so that this style only applies to children on actual `display-table` elements.

Less important but still worth pointing out, it's best to avoid ancestor selectors (`.total-row td`) and `!important`.

Ancestor selectors are broad and can end up applied in cases you don't expect.  Even worse, they can become a real performance-drain. In this case, for every single `td` element in the DOM, the browser needs to traverse backwards up the whole parent structure looking for a `total-row` class, and it won't stop looking till it reaches the root (then repeat on all applicable DOM updates). Modern browsers are decently optimized to be smart about it, but IE 11 does a real bad job of it.

`!important` should be reserved for an absolute last-option style overwrite (because it overpowers all others). This probably only needed to use it because the selector has "low specificity" (ancestor selectors are I think lowest specificity, another reason they're dicey). If you write an ugly long selector like `table.display-table > tbody > tr > td` then you probably won't need the `!important` any more.

Reply:
Oops, descendant selector, not ancestor selector I literally always make that mistake. Edited!